app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -131,6 +165,48 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
     [isFocused],
   )
 
+  const currentEventIndex = useSharedValue(-1)
+  const currentlyViewingEvents = useSharedValue<number[]>([])
+  const currentlyViewingSchedule = useSharedValue(0)
+
+  const handleViewableEventIndexChanged = useRef(
+    ({ viewableItems }: { viewableItems: RNViewToken[] }) => {
+      currentlyViewingEvents.value = viewableItems.map((item) => item.index)
+    },
+  ).current
+
+  const handleViewableScheduleIndexChanged = useRef(
+    ({ viewableItems }: { viewableItems: RNViewToken[] }) => {
+      currentlyViewingSchedule.value = viewableItems[0]?.index ?? 0
+    },
+  ).current
+
+  const scrollButtonOpacity = useDerivedValue(() => {
+    return withSpring(
+      scheduleIndex === currentlyViewingSchedule.value &&
+        currentEventIndex.value > -1 &&
+        currentlyViewingEvents.value[0] !== currentEventIndex.value
```

Comment:
So i think, if you change this index to 1 (`currentlyViewingEvents.value[1]`) it could solve the issue of the button still being visible even after you scroll to current. Since we figured out there's still an event "visible" just under the nav header, if we check for the 2nd one in the array instead of the first, it should register that the correct event truely is visible so the button should be hidden.

Reply:
Here's with index 0: 
![scroll-to-current-index-0](https://user-images.githubusercontent.com/6894653/217088378-0b50b839-af76-4ca1-9794-784620978d8a.gif)

And here's with index 1: 

https://user-images.githubusercontent.com/6894653/217088469-5043e1d1-eb69-4ab1-8eaa-f439f8884612.mp4

Reply:
So the only "issue" I see with setting the index to `1` is, it's fine when you click the button to go to the current event as you did on the second video. But if you scroll the screen, you see the button showing up -> skip one card -> then showing up again. If that's ok I can update the index to `1` :)

<video width="350" src="https://user-images.githubusercontent.com/53795920/217098259-35b5040b-be02-48dc-8515-50cbe6d5aed7.mp4">

Reply:
If I add
```
viewabilityConfig={{ itemVisiblePercentThreshold: 50 }}
```
to `FlashList` it shows up a bit later. the downside is you can see the arrow flickering again when I scroll slowly.

<video width="350" src="https://user-images.githubusercontent.com/53795920/217099227-46217b64-b25a-44d8-a551-63b5f8b9e645.mp4">

Reply:
Let's go with the `viewabilityConfig` solution as that feels the most correct to me, then we can file the flickering arrow as a bug.

Reply:
Sounds good 👍